### PR TITLE
[BRAAV-11383] Dynamically populate blockchain dropdown

### PR DESCRIPTION
**Pull request by elizalucas**
This PR dynamically populates the blockchain dropdown of the `POST /paymentintents` screen based on an api output. The flow of the page is as follows:

- when you navigate to the `POST /paymentIntents` page, only the currency dropdown is visible
- after selecting your desired currency the remaining fields of the page appear, and the blockchain dropdown is populated with the blockchains for the selected currency
- if you select a different currency, your selection in the blockchain dropdown clears and the items in the blockchain dropdown are repopulated based on the new currency selection

### CODE REVIEW

File: pages/debug/paymentIntents/create.vue [hunk 1]
```diff
@@ -86,17 +104,30 @@ export default class CreatePaymentIntentClass extends Vue {
...
   }
 
+  currencyBlockchainPairs: CurrencyBlockchainPairs =
+    this.$cryptoPaymentMetadataApi.getSupportedCurrencyAndBlockchainCombinations()
+
   required = [(v: string) => !!v || 'Field is required']
   error = {}
   loading = false
   showError = false
   supportedCurrencies = ['BTC', 'ETH', 'USD']
```
Comment (huaweigu) [inline pages/debug/paymentIntents/create.vue:114]: Do we actually need to hardcode `supportedCurrencies` initially? It should be dynamically populated from the api response

Comment (elizalucas) [reply pages/debug/paymentIntents/create.vue:114]: True - I will update this

Comment (elizalucas) [reply pages/debug/paymentIntents/create.vue]: Updated in this commit: [get currencies dynamically](https://github.com/circlefin/payments-sample-app/pull/242/commits/2df08982f74d74da4a8011b7f82c427095d5cbba)

**Review by huaweigu — APPROVED** [review]

File: pages/debug/paymentIntents/create.vue [hunk 2]
```diff
@@ -86,17 +104,33 @@ export default class CreatePaymentIntentClass extends Vue {
     expiresOn: '',
   }
 
+  currencyBlockchainPairs: CurrencyBlockchainPairs =
+    this.$cryptoPaymentMetadataApi.getSupportedCurrencyAndBlockchainCombinations()
```
Comment (kristinfritsch) [inline pages/debug/paymentIntents/create.vue:108]: I this missing await?

Comment (huaweigu) [reply pages/debug/paymentIntents/create.vue:108]: +1 
@elizalucas could it be the bug?

Comment (elizalucas) [reply pages/debug/paymentIntents/create.vue:108]: I think this has to do with it - I added this in [this PR ](https://github.com/circlefin/payments-sample-app/pull/246/files)but still not fully solved yet

**Pull request merged**